app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -82,13 +82,13 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
           await queryCirrusGraphQL(pr.head.sha, cirrusClient, log, 'flutter');
 
       // TODO(katelovett): flutter gold will need to support branching, https://github.com/flutter/flutter/issues/52700
-      if (!cirrusResults.any(
-          (CirrusResult cirrusResult) => cirrusResult.branch == 'master')) {
+      if (!cirrusResults.any((CirrusResult cirrusResult) =>
+          cirrusResult.branch == 'pull/${pr.number}')) {
         continue;
```

Comment:
Can you add a `log.debug(...);` here specifying that the PR is being skipped since no valid CirrusResult was found? That should make it easier to diagnose if there is ever an issue like this again.

Reply:
Added.